reporting.md

```diff
@@ -499,11 +497,11 @@ Figure: JSON Report Format
     Presently, the only three valid choices are `tlsa`, `sts`, and the literal
     string `no-policy-found`. It is provided as a string.
 * `policy-string`: The string serialization of the policy, whether TLSA record
-    or MTA-STS policy. Any linefeeds from the original policy MUST be replaced with
-    [SP]. TODO: Help with specifics.
-* `domain`: The Policy Domain upon which the policy was applied. For messages
-    sent to `user@example.com` this field would contain `example.com`. It is
-    provided as a string.
+    ([@!RFC6698] section 2.3) or MTA-STS policy. Any linefeeds from the original
+    policy MUST be replaced with space.
+* `domain`: The Policy Domain is the fully qualified DNS domain name ([@!RFC1035])
+    upon which the policy was applied ([@!RFC5890] for international domains). For
```

Comment:
*shrug* 

That's a very careful specification. We can keep it, but I'm not very clear on the wording as you have it. (Sorry!) But since we define "Policy Domain" in the "terminology" section, I would not neecssarily bother. I also do not believe we need to specify whether to use A form or U form; it matters for TLSA because they need to specify how to match the domain with the cert, but here it's just for reporting and recipients can presumably figure it out.